### PR TITLE
PRL-5346: Notifications for applicant LIP personal service

### PR DESCRIPTION
**Pull request by hcarslaw**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-5346

### Change description ###
Adds notifications for CA applicants after they are selected to serve the respondent personally

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/common/task-list/components/notification-banner/config/ca_applicant.ts [hunk 1]
```diff
@@ -83,4 +83,26 @@ export const CA_APPLICANT: NotificationBannerProps[] = [
       return caseData?.state !== State.CASE_CLOSED && !!caseData?.orderCollection?.length;
     },
   },
+  {
+    ...notificationBanner[BannerNotification.GIVE_RESPONDENT_THEIR_DOCUMENTS],
+    show: (caseData: Partial<CaseWithId>, userDetails: UserDetails): boolean => {
+      return (
+        caseData?.state === State.CASE_SERVED &&
```
Comment (vivek-sekhar) [inline src/main/steps/common/task-list/components/notification-banner/config/ca_applicant.ts:90]: Why shd case served be checked when the case is linked?

File: src/main/steps/common/task-list/components/notification-banner/index.ts [hunk 2]
```diff
@@ -39,35 +39,40 @@ export const getNotificationBannerConfig = (
...
                 noOfDaysRemainingToSubmitCase:
                   caseData?.noOfDaysRemainingToSubmitCase ?? 'caseData.noOfDaysRemainingToSubmitCase',
               }),
-            })),
-        };
+              html: content.html ?? null,
+            }));
+
+          const links = section?.links?.length
+            ? section?.links
```
Comment (vivek-sekhar) [inline src/main/steps/common/task-list/components/notification-banner/index.ts:56]: Why optional when you know the section has links already?

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/app/case/definition.ts [hunk 3]
```diff
@@ -196,6 +196,7 @@ export interface CitizenFlags {
   isAllegationOfHarmViewed?: string
   isAllDocumentsViewed?: string
   isResponseInitiated?: string
+  isApplicationToBeServed?: string
```
[the remark this answers is not in the document]
Comment (hcarslaw) [reply src/main/app/case/definition.ts:199]: Unable to change to use this as isApplicationToBeServed is a citizenFlag stored in the response and doesn't come directly in the case data

[the remark this answers is not in the document]
Comment (vivek-sekhar) [reply src/main/app/case/definition.ts:199]: ok no problem

File: src/main/steps/common/task-list/components/notification-banner/content.ts [hunk 4]
```diff
@@ -21,139 +21,251 @@ const en: NotificationBannerContent = {
...
+              ],
+            },
+            {
+              contents: [
+                {
+                  text: 'You can give the documents to the respondent or choose a person who has agreed to hand deliver them to the respondent. This can be someone you know or a professional third party (such as a process server or court bailiff). More information about court bailiffs can be found on GOV.UK.',
+                },
+                {
+                  text: '',
+                  html: '<a href="https://www.gov.uk/government/publications/form-d89-request-for-personal-service-by-a-court-bailiff">https://www.gov.uk/government/publications/form-d89-request-for-personal-service-by-a-court-bailiff</a>',
```
Comment (vivek-sekhar) [inline src/main/steps/common/task-list/components/notification-banner/content.ts:230]: Why html is required. Please use text property rather

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

**Review by vivek-sekhar — APPROVED** [review]

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**